### ISSUE TITLE
The length of test_data_loader is 0 (imagenet)

### ISSUE THREAD

**Issue by sspeedtail** [ZjjConan/Multi-Modal-Adapter#5]
I download the dataset as described in DATASETS.md. But when I run base2new_train.sh, it seems that the test set of imagenet can not be corrected loaded.  
File "train.py", line 202, in <module>
    main(args)
  File "train.py", line 137, in main
    trainer = build_trainer(cfg)
  File "/home/legion-w3/LabData/zxp/Dassl.pytorch/dassl/engine/build.py", line 11, in build_trainer
    return TRAINER_REGISTRY.get(cfg.TRAINER.NAME)(cfg)
  File "/home/legion-w3/LabData/zxp/Dassl.pytorch/dassl/engine/trainer.py", line 324, in __init__
    self.build_data_loader()
  File "/home/legion-w3/LabData/zxp/Dassl.pytorch/dassl/engine/trainer.py", line 347, in build_data_loader
    dm = DataManager(self.cfg)
  File "/home/legion-w3/LabData/zxp/Dassl.pytorch/dassl/data/data_manager.py", line 129, in __init__
    test_loader = build_data_loader(
  File "/home/legion-w3/LabData/zxp/Dassl.pytorch/dassl/data/data_manager.py", line 46, in build_data_loader
    assert len(data_loader) > 0
AssertionError
